components/Header/HeaderDashboard/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import styles from './index.module.scss';
+import { IconCrown, IconSetting, IconAddBox } from '@/assets/icongroup';
+
+export default function HeaderDashboard({ isOwner = true }) {
+  return (
+    <header className={`${styles['header-dashboard']}`}>
+      <div className={`${styles['header-dashboard-container']}`}>
+        <div className={`${styles['dashboard-title-container']}`}>

```

Comment:
styles 를 중복으로 사용하시는 게 아니면 className={styles['클래스명']} 으로 사용하실 수 있습니다! 
```suggestion 
className={{styles['header-dashboard']} 
```